content/about/privacy.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Privacy Policy"
+draft: false
+---
+InnerSource Commons is an independent non-for-profit organisation that creates and shares knowledge about InnerSource for its members. 
+
+This privacy policy will explain how our organisation uses the personal data we collect from you when you use our website and contribute to our projects.
+
+## What Data Do We Collect And Why?
+
+You directly provide InnerSource Commons with most of the data we collect. We collect and process data when you:
+
+- Register to our emailing list: InnerSource Commons collects personal identification information (Name and Email Address) to email you with news, events and articles we think you might like.
+
+- Register to our events: InnerSource Commons collects information that you provide us when you register for one of our events, such as conferences and when you choose to participate in an event as an attendee, speaker, or sponsor. This information includes personal data such as your name, company, and contact information.
+
+- Join our Slack community: When you join the conversation on Slack we store the information you contribute and your conversations.
```

Comment:
do we do that? or it's Slack's responsibility?

Reply:
Slack just provides us with a platform, the information on the platform is our responsibility.